### PR TITLE
Disable flaky test TestW3CSpanTraceResourceExtraction

### PR DESCRIPTION
**Pull request by dmitryax**
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9255

### CODE REVIEW

**Comment by dmitryax** [comment]
@pmm-sumo PTAL

**Review by jpkrohling — APPROVED** [review]

**Review by tigrannajaryan — APPROVED** [review]

File: exporter/awsxrayexporter/awsxray_test.go [hunk 1]
```diff
@@ -116,7 +117,8 @@ func constructSpanData() pdata.Traces {
 	return traces
 }
 
-func constructW3CSpanData() pdata.Traces {
+// nolint:unused
+func constructW3CSpanData() ptrace.Traces {
```
Comment (pmm-sumo) [inline exporter/awsxrayexporter/awsxray_test.go:121]: I think for now:

```suggestion
func constructW3CSpanData() pdata.Traces {
```

Comment (pmm-sumo) [reply exporter/awsxrayexporter/awsxray_test.go]: I have updated the code with this change @dmitryax to not hold disabling the test :)

**Review by pmm-sumo — APPROVED** [review]

File: exporter/awsxrayexporter/awsxray_test.go (hunk 1 above)
Comment (dmitryax) [reply exporter/awsxrayexporter/awsxray_test.go]: Oops. Thanks :)

**Comment by dmitryax** [comment]
@open-telemetry/collector-contrib-maintainer can you please help to merge this?

**Pull request merged**